src/procedures/procedures.11tydata.yaml

```
@@ -493,3 +493,59 @@ procedureMigrateConBee:
         <li class="info">All migrated devices have new entity IDs. This means you will need to manually reconfigure automations and dashboard cards.</li>
           <li class="info">The names that you manually assigned to your devices won't have been migrated. You will have to rename the devices again.</li>
         </ul>
+procedureEnableMultiProtocol:
+  title: Enable Multiprotocol support
+  steps:
+
+    - title: What is multiprotocol support?
+      image: skyconnect-multiprotocol-01.png
+      content: |
+        <ul>
+          <li class="warning">Multiprotocol is an experimental feature. It is not recommended that you use it just yet.</li>        
+          <li class="info">When multiprotocol support is enabled, the Home&nbsp;Assistant SkyConnect's IEEE&nbsp;802.15.4 radio can be used for both Zigbee and Thread (used by Matter) at the same time.</li>
+          <li>If the radio is already used by the ZHA Zigbee integration, ZHA will be reconfigured to use the multiprotocol firmware.</li>
+        </ul>
+    - title: Install the Silicon Labs Multiprotocol add-on
```

Installing the addon directly (not through the Hardware page) won't automatically re-configure ZHA to use it so I think we should just avoid mentioning it by name to discourage people from manually trying:

```suggestion
    - title: Enable Multiprotocol support
```